unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/Passport/Passport/PlayerPassportHUDView.cs

```diff
@@ -0,0 +1,36 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+using UnityEngine.Events;
+
+public class PlayerPassportHUDView : MonoBehaviour
```

Comment:
In order to follow our UI Components architecture, all our views should implement the `BaseComponentView` class.